pyproject.toml

```diff
@@ -43,6 +43,7 @@ sphinx = {version="*", optional=true}
 myst-parser = {version="^2.0.0", optional=true}
 pandas = {version=">=2.0.0,<3", optional=true}
 openpyxl = {version=">=3.0.7,<4", optional=true}
+ipython = "^8.15.0"
```

Comment:
I don't think this should be a required dependency. If you want to add it to the container that is fine, but it shouldn't be brought into our standard processing library.